librz/bin/format/elf/rz_bin_elf_init_phdr.inc

```diff
@@ -0,0 +1,135 @@
+// SPDX-FileCopyrightText: 2008-2020 nibble <nibble.ds@gmail.com>
+// SPDX-FileCopyrightText: 2008-2020 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2008-2020 alvaro_fe <alvaro.felipe91@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "elf.h"
+
+static const char *sdb_elf_p_type_cparse = "enum elf_p_type {PT_NULL=0,PT_LOAD=1,PT_DYNAMIC=2,"
```

Comment:
Unrelated to this PR but we should think about a better way to store those types instead of hardcoding it here. Maybe some additional data directory?